how-to/add-mods/modrinth-app.md

```diff
@@ -0,0 +1,15 @@
+---
+hidden: true
+---
+
+# Modrinth App
```

Comment:
Just noticed there is also a way to install via website now, so pls add separate instructions like CF
![pilt](https://github.com/user-attachments/assets/56fe87c1-bbe4-4c74-9835-55125d9953e3)

Reply:
```
1. Find the mod or mods you want to add from the [list of Fabric mods on Modrinth](https://modrinth.com/mods?g=categories:fabric)
2. Click on the mod details page, then **⤓ Download**
3. Click **◎ Install with Modrinth App ⍁**. The Modrinth App should open
4. Click **Install**
5. Next to the Fabulously Optimized instance, click **⤓ Install**
6. Close the popup with **×**
7. Once installed, go back to **|||\** (Library)
8. Launch the instance by hovering the icon and clicking **▷**. Minecraft should open up
9. If you can see the newly installed content in the Mod Menu, you're done!
```